src/main/java/org/idea/plugin/atg/visitor/XmlPsiRecursiveElementVisitor.java

```diff
@@ -0,0 +1,26 @@
+package org.idea.plugin.atg.visitor;
+
+import com.intellij.psi.PsiElement;
+import com.intellij.psi.PsiRecursiveElementVisitor;
+import com.intellij.psi.xml.XmlTag;
+
+import java.util.List;
+
+public class XmlPsiRecursiveElementVisitor extends PsiRecursiveElementVisitor {
```

Comment:
Name classes with self-described way. It's not clear that this class just filters all files where some tag is present